src/main/java/mekanism/common/item/gear/ItemFreeRunners.java

```diff
@@ -126,56 +116,13 @@ public boolean supportsSlotType(ItemStack stack, @NotNull EquipmentSlot slotType
 
     @Override
     public void adjustAttributes(ItemAttributeModifierEvent event) {
-        if (getMode(event.getItemStack()) == FreeRunnerMode.NORMAL) {
+        if (getMode(event.getItemStack()) == FreeRunnersMode.NORMAL) {
             event.addModifier(Attributes.MOVEMENT_EFFICIENCY, MOVEMENT_EFFICIENCY, EquipmentSlotGroup.FEET);
         }
     }
 
-    @NothingNullByDefault
-    public enum FreeRunnerMode implements IIncrementalEnum<FreeRunnerMode>, IHasEnumNameTextComponent, StringRepresentable {
```

Comment:
please at least restore the original enum name, it creates unnecessary noise in the diff